### ISSUE TITLE
Handle Imports

### ISSUE THREAD

**Issue by naomi-lgbt** [metz-sh/simulacrum#37]
I rarely, if ever, have a code file without any import statements. It would be nice if I could paste a file into the visualiser and it would "mock" the imports, so I could see the building blocks instead of a bunch of TS errors:

![image](https://github.com/user-attachments/assets/35fbda3f-8874-4159-99d6-ba34ac5e8763)

**Comment by iostreamer-X** [comment]
Fair point @naomi-lgbt. The lack of import actually stems from the fact that the compiler doesn't support "general" idiomatic TypeScript. There's quite an overlap, but there are a few things which the compiler needs in order to function.

That being said, the ability mock(in essence be flexible) rather than throw errors seems a great idea. Right now it gives a brittle feeling.

Thank you for bringing this up! Lemme start looking into this.
